docs/usage/authentication.md

```diff
@@ -0,0 +1,44 @@
+There are several ways to authenticate using the SDK:
+
+```python
+from pasqal_cloud import SDK
+
+project_id = "your_project_id"  # Replace this value with your project_id on the PASQAL platform. It can be found on the user-portal, in the 'project' section.
+username = "your_username"  # Replace this value with your username or email on the PASQAL platform.
+password = "your_password"  # Replace this value with your password on the PASQAL platform.
+```
```

Comment:
I think this snippet looks a bit odd without introduction; so maybe let's add a quick introduction like. "To authenticate with the SDK, you need to enter the credentials of your account as well as a project ID you are a member of. The jobs created with the SDK will then be billed to this project." Feel free to rephrase